README.md

```diff
@@ -15,10 +15,8 @@ Features:
 - [ ] Algorithm for game loop.
 
 Installing Packages using conda:
-- conda install -c anaconda pip
-- conda install -c selenium
-- conda install -c conda-forge opencv
+- conda install selenium
```

Comment:
This doesn't match with the commands given in the Anaconda website: https://anaconda.org/conda-forge/selenium
Does it still work?

Reply:
Unfortunately, "**conda install -c selenium**" does not work. It should either be "**conda install -c conda-forge selenium**" or "**conda install selenium**" for the package to install.